bftengine/src/simplestatetransfer/SimpleStateTran.cpp

```diff
@@ -89,6 +89,8 @@ class SimpleStateTran : public ISimpleInMemoryStateTransfer {
   void markUpdate(void* ptrToUpdatedRegion, uint32_t sizeOfUpdatedRegion) override;
   void setEraseMetadataFlag() override {}
 
+  virtual void handoffConsensusMessage(shared_ptr<ConsensusMsg>& msg) override{};
```

Comment:
See me prev. comment - should we have it as a pure interface or just define it in one of the lower classes?
Maybe we need ot discuss this.